controllers/nodefeaturediscovery_controls.go

```diff
@@ -79,22 +96,38 @@ func Namespace(n NFD) (ResourceStatus, error) {
 	return Ready, nil
 }
 
+// ServiceAccount checks if the ServiceAccount for NFD exists and attempts to
+// create it if it doesn't exist.
```

Comment:
Again, I'd just state that *creates*.

Hmm, actually should it state that we check the readiness of the resource or does it make sense (can serviceaccount not be ready). I.e. something like *ServiceAccount checks the readiness of the NFD ServiceAccount and creates it if it doesn't exist*, WDYT?